oci/layout/oci_transport.go

```diff
@@ -189,14 +191,28 @@ func (ref ociReference) NewImage(ctx *types.SystemContext) (types.Image, error)
 	return image.FromSource(src)
 }
 
-func (ref ociReference) getManifestDescriptor() (imgspecv1.Descriptor, error) {
+func (ref ociReference) getIndex() (imgspecv1.Index, error) {
+	index := imgspecv1.Index{
```

Comment:
(After the return values are modified to return `nil, err`, this should just an empty value.)

Reply:
?

Reply:
ahh,  I guess this relates to returning *imgspecv1.Index and nil in the case an error occurs. Fair enough

Reply:
This refers to https://github.com/containers/image/pull/326#discussion_r146985778 .